packages/cloud-cognitive/src/components/EditFullPage/EditFullPage.mdx

```diff
@@ -0,0 +1,227 @@
+import { Story, ArgsTable, Source, Canvas } from '@storybook/addon-docs';
+import {
+  getStoryId,
+  CodesandboxLink,
+} from '../../global/js/utils/story-helper';
+import { CreateFullPage } from '.';
+
+# EditFullPage
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Example usage](#example-usage)
+- [Class names](#class-names)
+- [Component API](#component-api)
+
+## Overview
+
+When to use
+
+- In Carbon for IBM Products, the Full page create flow is the process the user
+  must complete in order to use a product or service.
+- In **IBM Cloud public**, it is the alternative to the Provisioning flow for a
+  non-billable service.
+
+When not to use
+
+- Don’t use the Full page option when the product or service is usable before
+  the current creation flow is initiated.
+- For **IBM Public Cloud**, do not use this when provisioning a service directly
+  from the Catalog, or when creating a non-billable resource that could happen
+  in a side panel/in context.
```

Comment:
We should update this with [full page edit](https://pages.github.ibm.com/cdai-design/pal/patterns/edit/usage#full-page-edit)’s when to use section.